plat/kvm/x86/setup.c

```diff
@@ -40,6 +42,9 @@
 #include <uk/plat/console.h>
 #include <uk/assert.h>
 #include <uk/essentials.h>
+#include <x86/acpi/acpi.h>
+
+#include <uk/plat/lcpu.h>
```

Comment:
Why is this additional include required?

Reply:
Leftover from the first versions. Removed it